usaspending_api/common/experimental_api_flags.py

```diff
@@ -29,6 +32,13 @@ def is_experimental_elasticsearch_api(request: Request) -> bool:
     return request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
 
 
+def is_experimental_download_api(request: Request) -> bool:
```

Comment:
I would recommend sharing the same header value of `X-Experimental-API` (converted to `HTTP_X_EXPERIMENTAL_API` in the `request.META` by Django) and have different values `elasticsearch` vs `download`. You could then continue to have two different functions, but each would serve its own purpose based on the header value.

This is more of a functional programming approach, but for cases like this I think it is perfectly fine.